### PR TITLE
AP-3533: add final hearing merits question

### PR DESCRIPTION
**Pull request by colinbruce**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/AP-3533)

Add an additional, optional, question inside the proceeding loop at the start of the flow.
After the provider is shown the default Level of Service and Scope Limitations if they select No, they are asked to select a Level of Service.
If they choose Full Representation and it was not the default choice, we ask them to provide a date or reasons.
This gets repeated for both Emergency and Substantive work
It gets included on the CYA page (with no editing available yet, a separate ticket will address the whole CYA flow) and the review application pages
 

## Checklist

Before you ask people to review this PR:

- Tests and rubocop should be passing: `bundle exec rake`
- Github should not be reporting conflicts; you should have recently run `git rebase main`.
- There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- The PR description should say what you changed and why, with a link to the JIRA story.
- You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: app/controllers/providers/proceeding_loop/final_hearings_controller.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+      def update
+        @form = Proceedings::FinalHearingForm.new(form_params.merge(work_type:))
+        render :show unless save_continue_or_draft(@form, work_type:)
+      end
+
+    private
+
+      def work_type_param
+        sanitise(params.require(:work_type).html_safe)
```
[the remark this answers is not in the document]
Comment (colinbruce) [reply app/controllers/providers/proceeding_loop/final_hearings_controller.rb:19]: It's been too long a week to fight with this now... 
If a code reviewer wants to drop some pointers I'll be very grateful next week!

[the remark this answers is not in the document]
Comment (jsugarman) [reply app/controllers/providers/proceeding_loop/final_hearings_controller.rb:19]: Have not tried but just to be a rubber duck would it not be
```
sanitize(params.permit(:work_type)).html_safe
```

[the remark this answers is not in the document]
Comment (colinbruce) [reply app/controllers/providers/proceeding_loop/final_hearings_controller.rb:19]: ^ it's a view helper and not available in the controller! 🤦‍♂️ 
I said it was the end of a long week!

**Review by kmahern — APPROVED** [review]
lgtm

**Pull request merged**